src/CommunityToolkit.Maui.MediaElement/PlatformView/MediaManager.shared.cs

```diff
@@ -0,0 +1,207 @@
+﻿#if !(ANDROID || IOS || WINDOWS || MACCATALYST)
+global using PlatformMediaView = System.Object;
+#elif ANDROID
+global using PlatformMediaView = Com.Google.Android.Exoplayer2.IExoPlayer;
+#elif IOS || MACCATALYST
+global using PlatformMediaView = AVFoundation.AVPlayer;
+#elif WINDOWS
+global using PlatformMediaView = Microsoft.UI.Xaml.Controls.MediaPlayerElement;
+#endif
+
+using Microsoft.Extensions.Logging;
+
+namespace CommunityToolkit.Maui.MediaElement;
+
+/// <summary>
+/// An class that acts as a manager for an <see cref="IMediaElement"/> instance.
+/// </summary>
+public partial class MediaManager
+{
+	readonly IMauiContext mauiContext;
+	readonly IMediaElement mediaElement;
+#if ANDROID || IOS || MACCATALYST || WINDOWS || TIZEN
+	PlatformMediaView? player;
```

Comment:
Having those as `private` will hide from devs that want to customize the Manager, I suggest to change them to `protected`